.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--
+For high severity issues, please contact AWS Support directly.
+
+Please provide the following information:
+-->
+
+
+### Summary
+<!-- Please provide a brief outline of the issue -->
+
+
+### Description
+<!-- Provide detailed information about this issue -->
+
+
+
+<!--
+Please use these blocks to comment out appropriate sections
+especially for feature requests
+-->
+### Expected Behavior
+
+
+### Environment Details
+<!--
+Examples:
+* docker info
+* curl http://localhost:51678/v1/metadata
+* df -kh
+-->
+
+
+### Supporting Log Snippets
```

Comment:
It may be worthwhile to include a reminder that GitHub issues are public and that the customer should take care to exclude potentially sensitive data from logs before posting. We'll request additional logs via less public channels if necessary.